Contact Me Section

samlorlah opened the pull request:
### Contact Me

- New Contact Section added
- An header added to the section
- Description added to the section
- Contact form added
- Section and Form - flexbox style
- Contact form styled

samlorlah committed:
Add Contact Me Background image

Add Contact Me Comment to Style.css

Add contact-me section to index.html

juanmanuelbyc committed:
add form inputs and button to contact me section

samlorlah committed:
add action to contact me form

samlorlah committed:
add style to contact me section

juanmanuelbyc committed:
add style to button

samlorlah committed:
Linters Error for index.html and style.css corrected

1GORDON commented on index.html:
```diff
@@ -279,6 +279,22 @@ <h2 class="skill-name">Skills</h2>
         </div>
     </section>
-    
+    <section id="contact-me">
+        <h2 id="contact-me-header">Contact me</h2>
+        <p id="contact-me-description">If you have an application you are interested in developing, a feature that you need built or a project that needs
+        coding. I’d love to help with it</p>
+        <form id="contact-me-form" action="https://formspree.io/f/mpzbgqyj" method="post">
+            <div class="form-field">
+                <input required name="fullname" maxlength="30" type="text" placeholder="Enter your full name">
+            </div>
+            <div class="form-field">
+                <input required name="email" type="email" placeholder="Enter your email address">
+            </div>
+            <div class="form-field">
+                <textarea required name="message" maxlength="500" cols="30" rows="10" placeholder="Write your message here"></textarea>
```
Great job @samlorlah 🤝

- [ ] [Optional] You may please try to make your design as identical as possible. Please add that line at the bottom of your contact form.
- Your design
![your contact](https://user-images.githubusercontent.com/63917725/177372934-c865d60e-85df-4423-9b83-d9d528786201.PNG)

- Figma design
![figma contact](https://user-images.githubusercontent.com/63917725/177373008-afa830fa-3578-425c-a6d4-6466f1187753.PNG)

samlorlah replied:
@1GORDON .

Thanks for approving.

However your optional suggestion is noted but it is not needed. That is a symbol of home button for mobile view which comes naturally with phones like iPhones with Face ID.

Once again, I appreciate.

1GORDON reviewed (approved):
# Status: APPROVED 🟢 

## Hi @samlorlah ✋

<img src="https://media.giphy.com/media/BPJmthQ3YRwD6QqcVD/giphy.gif" width="480" height="270" class="giphy-embed" />

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! :tada: :clap: :clap: :100:

## Highlights!
- Good use of Gitflow ✔️ 
- Good commit messages  ✔️ 
- Professional Pull request ✔️
- Professional Readme ✔️
- Good UI ✔️

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@1GORDON` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

AbelG101 committed:
add a header and a paragraph

AbelG101 committed:
add styles for contact-me-header

add styles contact-me-description

add styles to contact-me-form

add styles to the form-field

add styles to the text-area

add styles to form input
